en/sim_gazebo_classic/README.md

```diff
@@ -470,6 +470,27 @@ Simply enter the following terminal command:
 gst-launch-1.0  -v udpsrc port=5600 caps='application/x-rtp, media=(string)video, clock-rate=(int)90000, encoding-name=(string)H264' \
 ! rtph264depay ! avdec_h264 ! videoconvert ! autovideosink fps-update-interval=1000 sync=false
 ```
+You can also customize the uri where the video stream is sent to, by [customizing the sitl_gazebo_classic parameters](jinja_override)
+
+
+### Customising the SITL Gazebo Classic Parameters
+
+SITL parameters can be specified for all vehicle instances or separately for every individual vehicle instance.
+These override the equivalent parameters in the main simulation jinja file.
```

Comment:
Originally this text was at the end and said "If any of these parameters is used in a jinja vehicle file, 
the parameters defined in the jinja file will be replaced by parameter in the jinja file. "

That's confusing. Can I assume that there is a "main jinja file" people will know about - so we're overwriting some data  that applies everywhere with some other data?

OR are you actually saying "The "all vehicle jinja file" sets all values. The individual jinja files overwrite these"? I.e there is no master file, you're explaining that the main file is used by default if there are missing individual vehicle files?

Reply:
@frederictaillandier Can you please check my review comments/edits.